### PR TITLE
<lesson 1.6>

### PR DESCRIPTION
**Pull request by ValeriGuerrero**

### CODE REVIEW

File: src/InputWithLabel.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import AddTodoForm from './AddTodoForm'
+import React from 'react';
+import { useEffect, useRef } from 'react';
+
+const InputWithLabel = ({ todoTitle, handleTitleChange, children, }) => {
```
Comment (slamflipstrom) [inline src/InputWithLabel.jsx:5]: nice use of destructuring on props here!

File: src/App.jsx [hunk 2]
```diff
@@ -24,14 +25,19 @@ const App = () => {
 
   };
 
+  const removeTodo = (id) => {
+    const newTodoList = todoList.filter((todo) => todo.id === id)
+    setTodoList(newTodoList)
+  }
+
```
Comment (slamflipstrom) [inline src/App.jsx:32]: I see that in running your code that the remove functionality isn't quite working correctly. Consider adding some more console.log statements to identify why the `todoList` isn't properly being filtered. 

I see the issue, and I'd be happy to answer any questions you have over Slack or in a 1:1 session. Feel free to ping me there if you want to chat more!

**Review by slamflipstrom — COMMENTED** [review]
You're close here, but the remove functionality still needs fixing

**Review by ValeriGuerrero — COMMENTED** [review]
I changed my code to !==

**Pull request merged**